api/nodes/views.py

```diff
@@ -3357,6 +3357,13 @@ class NodePreprintsList(JSONAPIBaseView, generics.ListAPIView, NodeMixin, NodePr
     view_category = 'nodes'
     view_name = 'node-preprints'
 
+    def postprocess_query_param(self, key, field_name, operation):
+        if field_name == 'provider':
+            operation['source_field_name'] = 'provider___id'
+
+        if field_name == 'id':
+            operation['source_field_name'] = '_guids___id'
```

Comment:
per wisecarver: "Those fields aren’t real fields they’re annotated onto the query specifically to reduce extra queries and are not guaranteed to be there." 

They do work and the filters do work properly locally. I'll swap to guids___id to be correct.